src/components/ChatEntry.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js';
+
 
 const ChatEntry = (props) => {
+
+  const likeMessage = () => {
+    const updatedEntry = {
+      ...props,
+      liked: !props.liked
+    }
+    props.onLike(updatedEntry)
+  }
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={`chat-entry ${props.remoteSend ? 'remote':'local'}`}>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time"><TimeStamp time={props.timeStamp}/></p>
+        <button className="like" onClick={() => likeMessage()}>{props.liked ? '❤️':'🤍'}</button>
```

Comment:
I'm not sure there's much point in passing the `() => likeMessage()` arrow function here compared to passing `likeMessage` itself.

It doesn't hurt anything, but it doesn't really do anything either.